contracts/generic/MultiSig.sol

```diff
@@ -94,7 +94,8 @@ contract MultiSig {
         script.state = ScriptState.Failed;
```

Comment:
You don't need to mark script as failed prior to executing it, because you 100% have enough gas to set correct `script.state` after `delegateCall`. This will save gas and allow to clean up lines 97-102. 

"problem" part of #10 is solved already, "code quality" isn't yet

Reply:
Good point, i'll remove the unnecessary parts.